x/oracle/abci.go

```diff
@@ -66,6 +66,10 @@ func MidBlocker(ctx sdk.Context, k keeper.Keeper) {
 			voteMapRD := ballotRD.ToMap()
 
 			exchangeRateRD := ballotRD.WeightedMedianWithAssertion()
+			// handle case where exchange rate is 0
+			if exchangeRateRD.IsZero() {
+				// TODO: do stuff
```

Comment:
do we actually want to do things?

Reply:
yeah I realized this one can't actually be zero because votes submitted with a zero value are modified to have the vote power set to 0, so the only way to get 0 exchange rates is for really small number divided by really large number which is with the cross rates calculations.